app/policies/peer_review_policy.rb

```diff
@@ -0,0 +1,9 @@
+# Policy for Peer reviews controller
+class PeerReviewPolicy < ApplicationPolicy
+  alias_rule :index?, :populate?, :assign_groups?, to: :assign_reviewers?
```

Comment:
You can change this one to a `default_rule`, and also rename the policy to the more general `manage?`.